### ISSUE TITLE
The New Features section of WCAG 2.2 has incorrect SC numbers on 3 SC

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag#3025]
In the [New Features section](https://www.w3.org/TR/WCAG22/#new-features-in-wcag-2-2), need to reorder and renumber the last 3 SC in the list to match the ordering and numbering in the document.
Instead of:
- 3.3.7 [Accessible Authentication](https://www.w3.org/TR/WCAG22/#accessible-authentication) (AA)
- 3.3.8 [Accessible Authentication (Enhanced)](https://www.w3.org/TR/WCAG22/#accessible-authentication-enhanced) (AAA)
- 3.3.9 [Redundant Entry](https://www.w3.org/TR/WCAG22/#redundant-entry) (A)

It should be:
- 3.3.7 [Redundant Entry](https://www.w3.org/TR/WCAG22/#redundant-entry) (A)
- 3.3.8 [Accessible Authentication](https://www.w3.org/TR/WCAG22/#accessible-authentication) (AA)
- 3.3.9 [Accessible Authentication (Enhanced)](https://www.w3.org/TR/WCAG22/#accessible-authentication-enhanced) (AAA)

**Comment by patrickhlauke** [comment]
duplicate of https://github.com/w3c/wcag/issues/2975 which has been actioned (and will appear correctly next time 2.2 is republished, I believe)

**Comment by maryjom** [comment]
@patrickhlauke Thanks. I couldn't find it through my search.

**Comment by patrickhlauke** [comment]
no worries, i only remembered because it was fresh in my mind from recent discussions :)
